app/Models/Sensor.php

```diff
@@ -38,6 +38,7 @@ class Sensor extends DeviceRelatedModel implements Keyable
         'group',
         'rrd_type',
     ];
+
     protected static $icons = [
```

Comment:
Since this is the source now as I understand it, we should move this to a enum class

Reply:
Done